### PR TITLE
Swapping receivers for two var graph

### PR DESCRIPTION
**Pull request by maxwellfundi**
@dannyparsons  I did swap the receivers as needed in #3836  so its ready to merge.

### CODE REVIEW

File: instat/dlgDescribeTwoVarGraph.Designer.vb [hunk 1]
```diff
@@ -93,22 +93,10 @@ Partial Class dlgDescribeTwoVarGraph
...
-        Me.ucrReceiverMultipleTwoVar.strNcFilePath = ""
-        Me.ucrReceiverMultipleTwoVar.TabIndex = 1
-        Me.ucrReceiverMultipleTwoVar.ucrSelector = Nothing
-        Me.ucrReceiverMultipleTwoVar.ucrVariableSelector = Nothing
-        '
         'ucrSecondVariableReceiver
         '
         Me.ucrSecondVariableReceiver.frmParent = Me
-        Me.ucrSecondVariableReceiver.Location = New System.Drawing.Point(261, 191)
+        Me.ucrSecondVariableReceiver.Location = New System.Drawing.Point(264, 58)
```
Comment (lilyclements) [inline instat/dlgDescribeTwoVarGraph.Designer.vb]: Can you move this to a height of 60?

File: instat/dlgDescribeTwoVarGraph.Designer.vb [hunk 2]
```diff
@@ -62,11 +62,11 @@ Partial Class dlgDescribeTwoVarGraph
         'lblSecondVariable
         '
         Me.lblSecondVariable.AutoSize = True
-        Me.lblSecondVariable.Location = New System.Drawing.Point(261, 176)
+        Me.lblSecondVariable.Location = New System.Drawing.Point(261, 105)
```
Comment (lilyclements) [inline instat/dlgDescribeTwoVarGraph.Designer.vb]: This needs to move down by 3 too to a height of 108

**Comment by dannyparsons** [comment]
@maxwellfundi this seems like something that would be good to sort out before I leave.

**Comment by maxwellfundi** [comment]
@lilyclements  you are right. this is the dialog @volloholic  said the code was very inefficient and needs redoing. 
We could actually discuss this with @dannyparsons before he leaves.

**Comment by dannyparsons** [comment]
@maxwellfundi can you specify the issues that we need to discuss? Or start working on them yourself?

**Comment by maxwellfundi** [comment]
@dannyparsons  this dialog needs re-writting to make sure the code is efficient enough because in its current state its not. all the same the first task was just to swap receivers then the code would be discussed.

**Comment by dannyparsons** [comment]
Ok, so if this has been swapped to do it in the way we now want it would be good to merge this and then the rewriting is a separate task. Should someone now review this?

**Comment by maxwellfundi** [comment]
yes, @shadrackkibet  can you help review this?

**Comment by shadrackkibet** [comment]
I think it will be good if you make lily's reviews then it will be ready for @dannyparsons .

**Comment by maxwellfundi** [comment]
Done!

**Comment by shadrackkibet** [comment]
@dannyparsons this is ready for you.

**Comment by lilyclements** [comment]
Not all my suggestions have been addressed - for example, I think you still need to change the now-top receiver to be `.SetMeAsReceiver`. (Not sure if there's other suggestions I made to be done still! I just got an email of this conversation and had a spare 2 minutes to check...)

**Comment by dannyparsons** [comment]
Did you make the top receiver the current receiver? I only see changes on the designer code.

**Comment by maxwellfundi** [comment]
Thus should be fine now

File: instat/dlgDescribeTwoVarGraph.vb [hunk 3]
```diff
@@ -118,7 +118,7 @@ Public Class dlgDescribeTwoVarGraph
         ucrSaveGraph.Reset()
         ucrSelectorTwoVarGraph.Reset()
 
-        ucrReceiverMultipleTwoVar.SetMeAsReceiver()
+        ucrSecondVariableReceiver.SetMeAsReceiver()
```
Comment (dannyparsons) [inline instat/dlgDescribeTwoVarGraph.vb]: This should have a different name now, maybe FirstVariableReceiver.

**Comment by maxwellfundi** [comment]
Closing this because it has conflicts, will put up a new fresh one.

**Pull request closed without merging**